src/frontend/lib/file-system.ts

```diff
@@ -0,0 +1,3 @@
+import {ExternalDirectoryPath} from 'react-native-fs';
+
+export {ExternalDirectoryPath as EXTERNAL_FILES_DIR};
```

Comment:
Do we have a plan for iOS? This is only supported by Android

Reply:
no plan for now, as i'm not super familiar with the filesystem apis for iOS. my guess is that it would be what react-native-fs calls the `DocumentDirectoryPath` (https://developer.apple.com/documentation/foundation/nssearchpathdirectory/nsdocumentdirectory). happy to add this in if you think it's worth doing now vs later